lib/config.coffee

```diff
@@ -91,6 +90,10 @@ exports.generate = (options, params = {}) ->
 		apiKey: options.apiKey
 
 		connectivity: params.connectivity or 'connman'
+
+	if not options.version or options.version < '2.0.0'
```

Comment:
@zvin its might be good to checkout with @aghezan and @petrosagg if the resinOS versioning scheme adheres to semver, I think it might not exactly conform :/